### PR TITLE
fix(containerize): remove autoconfig for buildctl

### PR DESCRIPTION
**Pull request by aarnphm**
People who use buildctl should probably know how to use it.

Remove an auto --output from tag

Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

### CODE REVIEW

**Comment by sauyon** [comment]
How many releases has this been a thing? Can we leave it in unless the option is specifically passed?

**Comment by sauyon** [comment]
@ssheng just an FYI that this would be a breaking change, but talking to Aaron I think this makes sense.

**Comment by ssheng** [comment]
Thanks, @aarnphm and @sauyon. What was the motivation of this change?

**Comment by aarnphm** [comment]
cc @ssheng https://github.com/bentoml/BentoML/issues/3483

File: src/bentoml_cli/containerize.py [hunk 1]
```diff
@@ -540,14 +540,27 @@ def containerize(  # type: ignore
                         "To load image built with 'buildctl' requires one of docker, podman, nerdctl (Neither are found in PATH) and try again."
                     )
                     sys.exit(0)
-                o = subprocess.check_output([container_runtime, "load"], input=result)
-                click.echo(o.decode("utf-8").strip())
+                # NOTE: We will always use the docker image spec if docker is available.
+                # Otherwise fallback to the OCI image spec.
+                if "output" not in _memoized:
```
Comment (sauyon) [inline src/bentoml_cli/containerize.py:545]: What's this change do?

**Review by sauyon — COMMENTED** [review]
o

File: src/bentoml_cli/containerize.py (hunk 1 above)
Comment (aarnphm) [reply src/bentoml_cli/containerize.py:550]: deprecation warning right?. I also add the instruction for loading the tar into local container registry.

Comment (sauyon) [reply src/bentoml_cli/containerize.py:550]: I'm not sure why we need to warn twice if you haven't specified output?

Comment (aarnphm) [reply src/bentoml_cli/containerize.py:550]: We need to do that right. First warning is to say the autoconfig is deprecated. The second one is instruct how they can do it with passing in output.

**Review by ssheng — APPROVED** [review]

**Pull request merged**